src/modules/app/actions/setUIDirection.js

```diff
@@ -0,0 +1,10 @@
+// @flow
```

Comment:
Why did you put the uiDirection in the redux state? You could also access the context in the CategoriesPage and use 
```js
RTL_LANGUAGES.includes(targetLanguage) ? 'rtl' : 'ltr'
```
there

Reply:
Hmm.. I think its up to our i18nProvider to decide which language is rtl 🤔

Reply:
Maybe a static method would do the job?

Reply:
I think its ok to have it like it is. But a i18n module would be nice maybe.

Reply:
Hmm... yeah that's right... I'lll move I18nProvider along to the i18n package

Reply:
Done.